pkg/xen/Dockerfile

```diff
@@ -57,9 +57,18 @@ RUN case $(uname -m) in \
     make defconfig && \
     make oldconfig
 
-RUN mkdir -p /out/boot
+RUN make 
 
-RUN make && cp xen.gz /out/boot/xen.gz
+RUN case $(uname -m) in \
+    x86_64) \
+	mkdir -p /out/boot; \
+	cp xen.gz /out/boot/xen.gz; \
+	;; \
+    aarch64) \
+	mkdir -p /out/boot/efi/EFI; \
+	cp xen /out/boot/efi/EFI/xen; \
```

Comment:
So, if I understand this right, the (grub?) bootloader for x86_64 that loads xen expects the compressed xen binary to be in boot/xen, while the bootloader for aarch64 expects the uncompressed xen binary to be in boot/efi/EFI/xen, and we couldn't have stuck with the same format and same path for aarch64? What is the bootloader being used for aarch64?

Reply:
Xen ARM output binary can also be run as an EFI. xen.efi can be built in x86 but requires a PE-supporting toolchain. Also we don't use it.

I think that the discussion of booting in ARM is still up in the air given how boards vary from each other.